tools-iuc/test-data.yaml

```diff
@@ -0,0 +1,3058 @@
+- data:
+    operation_type|input_reads:
+      filetype: null
+      location: https://raw.githubusercontent.com/galaxyproject/tools-iuc/master/tools/meryl/test-data/child.fasta
+      sha: 5223a9704fb4547ed6f1afd7f5da3e2c03531ab2
```

Comment:
is sha256 stronger?

Reply:
so far I was just getting the `sha`s from the github API: https://api.github.com/repos/galaxyproject/tools-iuc/contents/tools/purge_dups/test-data/split_out.fasta

to get sha256 I would have to download all the data :sweat: but I can do it of course

Reply:
Ah I see ... no lets leave this for now. We can also calculate other hashs via CI later.